daffodil-propgen/src/main/scala/org/apache/daffodil/propGen/TunableGenerator.scala

```diff
@@ -91,11 +91,11 @@ class TunableGenerator(schemaRootConfig: scala.xml.Node, schemaRootExt: scala.xm
   val middle = """
     |  extends Serializable {
     |
-    |  def setTunables(tunables: Map[String, String]): DaffodilTunables = {
-    |    tunables.foldLeft(this) { case (dafTuns, (tunable, value)) => dafTuns.setTunable(tunable, value) }
+    |  def withTunables(tunables: Map[String, String]): DaffodilTunables = {
+    |    tunables.foldLeft(this) { case (dafTuns, (tunable, value)) => dafTuns.withTunable(tunable, value) }
     |  }
     |
-    |  def setTunable(tunable: String, value: String): DaffodilTunables = {
+    |  def withTunable(tunable: String, value: String): DaffodilTunables = {
```

Comment:
This functions aren't technically deprecated, but this rename is semi-related to other deprecated functions and is a good change--the generated `setTunable` functions are immutable and so should use the same `with*` naming convention :+1: